A few tests require downloading models. Skip those.

diff --git a/onnx/backend/test/runner/__init__.py b/onnx/backend/test/runner/__init__.py
index c5ce045d..de241809 100644
--- a/onnx/backend/test/runner/__init__.py
+++ b/onnx/backend/test/runner/__init__.py
@@ -245,6 +245,7 @@ class Runner:
                     f"Start downloading model {model_test.model_name} from {model_test.url}"
                 )
                 filename = os.path.join(tmpdir, "file")
+                raise unittest.SkipTest("Skipping download")
                 urlretrieve(model_test.url, filename)
                 print("Done")
                 onnx.utils._extract_model_safe(filename, models_dir)
